isic/settings.py

```diff
@@ -114,6 +115,7 @@ class DevelopmentConfiguration(IsicMixin, DevelopmentBaseConfiguration):
     CELERY_TASK_ALWAYS_EAGER = values.BooleanValue(False)
     CELERY_TASK_EAGER_PROPAGATES = values.BooleanValue(False)
     ISIC_DATACITE_DOI_PREFIX = '10.80222'
+    HASHID_SALT = values.Value('deadbeef')
```

Comment:
Unless there's a need to overwrite this from the environment, it can just be a literal, not a `values.Value`.